TINY-11932: Changelog for 7.7.2 (#10223)
tinymce/tinymce: .changes/tinymce/7.7.2.md

> ## 7.7.2 - 2025-03-17

Reviewer: ```suggestion
## 7.7.2 - 2025-03-19
```

### Fixed

> - Exception when pressing tab in the last cell of a non-editable table. #TINY-11797
> - Exception was thrown when trying to use context form API after component was detached. #TINY-11781
> - Deleting an empty block within <li> would move cursor at the end of the <li>. #TINY-11763
> - Deleting an empty block that is between two lists would throw an error when all of the elements were nested inside the same <li>. #TINY-11763

Reviewer 2: ```suggestion
- Error was thrown when pressing tab in the last cell of a non-editable table. #TINY-11797
- Error was thrown when trying to use the context form API after a component was detached. #TINY-11781
- Deleting an empty block within an <li> element would move cursor to the end of the <li>. #TINY-11763
- Deleting an empty block that was between two lists would throw an Error when all three elements were nested inside a list. #TINY-11763
```
I didn't see that last change entry at the time, the wording of the fix was not quite accurate. cc @kemister85

Reviewer: LGTM, ill update the docs entries on our end.
